lib/bambora/factories/response_adapter_factory.rb

```diff
@@ -8,8 +8,11 @@ class << self
       def for(response)
         content_type = response.headers['Content-Type']
         case content_type
-          when 'application/json' then Bambora::JSONResponse.new(response)
-          when 'text/html' then Bambora::QueryStringResponse.new(response)
+          when 'application/json'
+            Bambora::JSONResponse.new(response)
+          when 'text/html'
+            # Currently, the only endpoint that responds wit text/html is /scripts/payment_profiles.asp
```

Comment:
As we build other endpoints, we might need to change the case statement here to look at request URLs instead of content type. But for now, this works fine.

Reply:
I'm all for this iterative approach.